Sources/StreamChatUI/Gallery/Cells/VideoAttachmentGalleryCell.swift

```diff
@@ -56,6 +56,11 @@ open class VideoAttachmentGalleryCell: GalleryCollectionViewCell {
                 AVPlayerItem(asset: components.videoLoader.videoAsset(at: $0))
             }
             player.replaceCurrentItem(with: playerItem)
+
+            NotificationCenter.default.addObserver(self,
```

Comment:
@mdhanif-simformsolutions we should use `NotificationCenter.default.addObserver` in `setUpLayout` method.

Reply:
@parth-simformsolutions Updated this. Thank you.